data/linux-bridge/003-bridge-marker-rbac.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: bridge-marker-cr
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - nodes/status
+  verbs:
+  - get
+  - update
+  - patch
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: bridge-marker-crb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: bridge-marker-cr
+subjects:
+- kind: ServiceAccount
+  name: bridge-marker
+  namespace: linux-bridge
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: bridge-marker
+  namespace: linux-bridge
+{{ if .EnableSCC }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: bridge-marker
+allowHostNetwork: true
+allowHostPorts: true
```

Comment:
Why do we need this?

Reply:
Yes, the pod will not be scheduled without allowHostPorts.

Reply:
I wonder why. Could you please share events from the DS and respective replica set? You should find them in the `kubectl describe ds bridge-marker` and `kubectl describe replicaset bridge-marker` (or something similar, not sure if it gets suffix).

Reply:
Or just point me to the place in manifests where we request host ports.

Reply:
tried this out again. Indeed it works without allowHostPorts (why did it not previously?)
Pushed an update.

Reply:
Thanks :)